src/app/features/high-fee-drawer/high-fee-drawer.tsx

```diff
@@ -18,8 +18,7 @@ export function HighFeeDrawer(props: { learnMoreUrl: string }) {
 
   return (
     <ControlledDrawer
-      // #4476 FIXME update this icon
-      icon={<FiAlertTriangle color="error.label" size="icon.xl" />}
+      icon={<ErrorIcon color="error.label" size="icon.xl" />}
```

Comment:
The shared icons do need to use the `token()` import to pass tokens or the styling doesn't seem to work.

Reply:
The compiled source is helpful debugging Panda issues. Could be wrong, but my read of this is:

```ts
export interface SquareProperties {
   size?: PropertyValue<'width'>
}
```

square accepts properties that are defined within the width property, which allows these values:

```
	width: "auto" | Tokens["sizes"] | "1/2" | "1/3" | "2/3" | "1/4" | "2/4" | "3/4" | "1/5" | "2/5" | "3/5" | "4/5" | "1/6" | "2/6" | "3/6" | "4/6" | "5/6" | "1/12" | "2/12" | "3/12" | "4/12" | "5/12" | "6/12" | "7/12" | "8/12" | "9/12" | "10/12" | "11/12" | "screen";
```

The icon sizes you've created are under a custom token name. If they were defined as `sizes` can we use the props directly?

Reply:
Looking into 
I debugged on the `SendForm` and the error `Error: <svg> attribute width: Expected length` comes from `Error: <svg> attribute width: Expected length` which gets loaded in the DOM but unseen.

Reply:
Yeah, was going to revisit this setup, but the design team is also looking at using an icon library which might be the easiest path.

Reply:
After looking at the docs further, I'm not sure our `spacing` tokens follow the format they suggest either? https://panda-css.com/docs/theming/tokens#spacings

Why did we use `space.00`, etc? Not sure I was working on the UI project when that decision was made.

Reply:
We discussed it at length at the time and we made a decision to go with number based spacing as a lot of other people use that and it seems standard. Based on research by @kyranjamie

Reply:
Panda comes with defaults you can use, shown in the docs, but my read of this isn't that panda suggests these and that's defacto what you're supposed to do. It's configurable to work with any design system. [Here the panda docs use number spacing units.](https://panda-css.com/docs/customization/theme#spacing)

There's no inherent issue with `token()`, you need it in many cases (external libs etc). But afaik it doesn't work here because we're trying to use the wrong type spacing units.

If we define the icon sizes in sizes, the problem is fixed?

Separately though, are we sure we want to wrap all the icons in Square? Why do we do this? What if we want to customise the underlying svg? `styled.svg` will accept any tokens, also fixing the token/string issue?

Reply:
What I've come up with in my last commit appears to work best so far. Can you all review it? I am wrapping it in `Square` bc it seemed to be the only way to pass `size` prop or you often have to wrap it in a Box or Square anyway in the component where it is used to change the size, is that not correct bc that is the behavior I was seeing in my initial refactor PR so I thought best just to do it consistently and allow for the `size` prop to always be an option (with a default set in the icon component). We can use `width`, but it didn't seem as intuitive to me as using `size`. Happy to change though.